source/features/show-names.tsx

```diff
@@ -40,8 +40,8 @@ async function init() {
 		if (name) {
 			// If it's a regular comment author, add it outside <strong>
 			// otherwise it's something like "User added some commits"
-			const insertionPoint = usernameEl.parentNode.tagName === 'STRONG' ? usernameEl.parentNode : usernameEl;
-			insertionPoint.after(' (', <bdo>{name}</bdo>, ') ');
+			const insertionPoint = usernameEl.parentElement.tagName === 'STRONG' ? usernameEl.parentNode : usernameEl;
+			(insertionPoint as HTMLElement).after(' (', <bdo>{name}</bdo>, ') ');
```

Comment:
I think this type is being asked here because on the line above there’s still a `parentNode`.

Reply:
I'll take a peek.

Reply:
Fixed.